src/components/ui/toasts/ToastError.tsx

```diff
@@ -3,28 +3,48 @@
 import { useContext, useEffect, useState } from 'react';
 import Alert from '../icons/Alert';
 import { MapDataContext } from '@/components/contexts/MapDataProvider';
+import { ErrorContext } from '@/components/contexts/ErrorProvider';
 
-interface Props {
-	title: string;
-	description?: string;
-	className?: string;
-}
-
-export default function ToastError({ title, description, className }: Props) {
+export default function ToastError() {
 	const [isVisible, setIsVisible] = useState(true);
 	const { isSidebarOpen } = useContext(MapDataContext);
+	const { errorContent } = useContext(ErrorContext);
+	const [title, setTitle] = useState('');
+	const [description, setDescription] = useState('');
 	useEffect(() => {
 		const timer = setTimeout(() => {
 			setIsVisible(false);
 		}, 2500);
 
 		return () => clearTimeout(timer);
 	}, []);
+	useEffect(() => {
+		switch (errorContent) {
+			case 'filter':
+				setTitle('한 개 이상의 필터를 선택해주세요');
+				setDescription('');
+				return;
+			case 'search':
+				setTitle('검색 결과가 없습니다');
+				setDescription('검색어를 다시 확인해주세요');
+				return;
+			case 'seoul':
+				setTitle('더 이상 조회할 수 없습니다');
+				setDescription('지금은 서울시의 수거함만 조회할 수 있어요');
+				return;
+			case 'data':
+				setTitle('수거함 정보가 없습니다');
+				setDescription('더 많은 정보를 불러올 수 있도록 준비 중이에요');
+				return;
+			case 'review':
+				setTitle('방문 기록을 등록할 수 없습니다');
+				setDescription('방문 기록은 24시간 이후에 재등록할 수 있어요');
+		}
+	}, [errorContent]);
```

Comment:
이렇게 수정하는 방법도 있을 것 같습니다.

```suggestion
const ERROR_MESSAGES = {
    filter: { title: '한 개 이상의 필터를 선택해주세요', description: '' },
    search: { title: '검색 결과가 없습니다', description: '검색어를 다시 확인해주세요' },
    seoul: { title: '더 이상 조회할 수 없습니다', description: '지금은 서울시의 수거함만 조회할 수 있어요' },
    data: { title: '수거함 정보가 없습니다', description: '더 많은 정보를 불러올 수 있도록 준비 중이에요' },
    review: { title: '방문 기록을 등록할 수 없습니다', description: '방문 기록은 24시간 이후에 재등록할 수 있어요' }
};

useEffect(() => {
    const errorMessage = errorMessages[errorContent];
    setTitle(errorMessage.title);
    setDescription(errorMessage.description);
}, [errorContent]);
```

다시보니 setTitle과 setDescription을 하나의 상태로 합쳐도 괜찮겠네요